src/app/services/dns-check.service.ts

```diff
@@ -2,19 +2,22 @@ import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
 import {AlertService} from './alert.service';
 import {AppService} from './app.service';
+import { TranslateService } from '@ngx-translate/core';
 
 @Injectable()
 export class DnsCheckService {
   private backendUrl: string;
   private clientInfo: object;
   private _profiles: string[];
 
-  constructor(private alertService: AlertService, private http: HttpClient) {
+  constructor(private alertService: AlertService, 
+    private http: HttpClient,
+    private translateService: TranslateService) {
     this.backendUrl = AppService.apiEndpoint();
     this.clientInfo = AppService.getClientInfo();
 
     if (this.backendUrl) {
-      console.error('Please set the api endpoint');
+      console.error(this.translateService.get('Please set the api endpoint'));
```

Comment:
Why using a different syntax here ? And not, as in the other files, something like :
```
this.translateService.get('Please set the api endpoint').subscribe((res: string) => {
  console.error(res);
});
```